lib/sidekiq/manager.rb

```diff
@@ -134,12 +134,18 @@ def real_thread(proxy_id, thr)
       @threads[proxy_id] = thr
     end
 
+    PROCTITLES = [
+      Proc.new { 'sidekiq'.freeze },
+      Proc.new { Sidekiq::VERSION },
+      Proc.new { |mgr, data| data['tag'] },
+      Proc.new { |mgr, data| "[#{mgr.busy.size} of #{data['concurrency']} busy]" },
+      Proc.new { |mgr, data| "stopping" if mgr.stopped? },
```

Comment:
it's style preference but may be will we use `proc` instead `Proc.new` and `x.(arg1, arg2)` instead `x.call(arg1, arg2)`?